### PR TITLE
Fix:2143 Modified comment for the style tag

### LINKED ISSUE #2143: link tags do not need to go in the head
adding a link snippet for a css file says 
```
    <!-- Add the <link> tag to the <head> tag of your HTML file -->
    <link href="style.css" rel="stylesheet">
```

but there is no need to do this. It's good form, certainly, but link tags can go anywhere and just work(tm)

### PR DESCRIPTION
**Pull request by Rajat-dhyani**
Fixes #2143

### CODE REVIEW

File: server/lib/snippets.js [hunk 1]
```diff
@@ -109,7 +109,7 @@ const html = [{
   title: "snippetHTMLExternalStylesheetTitle",
   data: {
     value:
-`<!-- {{ gettext("snippetHTMLExternalStylesheetComment") | safe }} -->
+`<!-- {{ gettext("snippetHTMLExternalStyleSheetComment") | safe }} -->
```
Comment (flukeout) [inline server/lib/snippets.js]: I dont understand why this is showing up as a change, is anything different here?

Comment (Rajat-dhyani) [inline server/lib/snippets.js]: I have capitalized second 's'  of   Style-**S**-heet so as to make it a new key and we can then localize it..

File: locales/en-US/messages.properties [hunk 2]
```diff
@@ -112,7 +112,7 @@ snippetHTMLScript=Include a JavaScript file
 snippetHTMLScriptTitle=Add a Script
 snippetHTMLExternalStylesheet=Include a CSS file
 snippetHTMLExternalStylesheetTitle=Add an External Stylesheet
-snippetHTMLExternalStylesheetComment=Add the <link> tag to the <head> tag of your HTML file
+snippetHTMLExternalStyleSheetComment=Add the <link> tag to your HTML file, preferably to the <head> tag
```
Comment (Pomax) [inline locales/en-US/messages.properties]: I'd recommnd `Add the <link> tag to your HTML file.  These usually go in the <head> tag`

Comment (Rajat-dhyani) [inline locales/en-US/messages.properties]: done :)

**Comment by flukeout** [comment]
Thanks very much @Rajat-dhyani 💯 👍

**Pull request merged**